test/browser/delay.js

```diff
@@ -0,0 +1,18 @@
+var delay = 500;
+var start = new Date().getTime();
+
+setTimeout(function() {
+  describe('delayed execution', function() {
+    it('should have waited ' + delay + 'ms to run this suite', function() {
+      assert(new Date().getTime() - delay >= start);
+    });
+
+    it('should have no effect if attempted twice in the same suite', function() {
+      assert(true);
+      run();
+      assert(true);
```

Comment:
Maybe this spec could be updated? Rather than the two ```assert(true)```, we could have another assertion on the time, ensuring that some number less than delay has elapsed since (start + delay)